b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/data/templates/enterprise-identity-provider.json

```diff
@@ -0,0 +1,142 @@
+                            "description": "The client identifier value of the identity provider.",
+                            "type": "STRING",
+                            "displayOrder": 1,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "ClientSecret",
+                            "displayName": "Client Secret",
+                            "description": "The client secret value of the identity provider.",
+                            "type": "STRING",
+                            "displayOrder": 2,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": true,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "OAuth2AuthzEPUrl",
+                            "displayName": "Authorization Endpoint URL",
+                            "description": "The standard authorization endpoint URL obtained from the identity provider.",
+                            "type": "STRING",
+                            "displayOrder": 3,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "OAuth2TokenEPUrl",
+                            "displayName": "Token Endpoint URL",
+                            "description": "The standard token endpoint URL obtained from the identity provider.",
+                            "type": "STRING",
+                            "displayOrder": 4,
+                            "regex": ".*",
+                            "isMandatory": true,
+                            "isConfidential": false,
+                            "options": [],
+                            "defaultValue": "",
+                            "subProperties": []
+                        },
+                        {
+                            "key": "callbackUrl",
+                            "displayName": "Callback Url",
+                            "description": "The WSO2 Identity Server URL to which the user needs to be redirected after completing the authentication at the identity provider. The identity provider needs to send the authorization code to this URL upon successful authentication.",
```

Comment:
Please check this line doesn't exceed the line length

Reply:
This is a template file, and IMO it is okay to exceed 120 characters in such files, as we have done the same in our other repos as well.

ex: 
- https://github.com/wso2-enterprise/asgardeo-apps/blob/58c7f91640fae55d4215c33a7075399aeabe7fe2/apps/console/src/extensions/identity-provider-templates/templates/saml/saml.json#L61